keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -645,10 +653,14 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Give a driver that does not ControllerModifyVolume, CSI volume should not be modified.
 - If ControllerModifyVolume fails, PVC should have appropriate events.
 
+[Tests PR](https://github.com/kubernetes/kubernetes/pull/121849)
+
```

Comment:
The referenced test PR is still a draft?

Reply:
Hi @carlory, @sunnylovestiramisu. 

I see all comments on [VAC e2e test PR #121849](https://github.com/kubernetes/kubernetes/pull/121849) are all resolved or answered. Are we ready to un-draft?

Reply:
Offline discussed that only E2E csi test must be merged for KEP merge.